Support/cmake/p4est.cmake

```diff
@@ -124,7 +124,9 @@ macro(p4est_build)
     INSTALL_COMMAND     cd ${CMAKE_SOURCE_DIR}/ThirdParty/p4est && make install --silent V=0
   )
   add_dependencies(p4est zlib)
-  add_dependencies(p4est openblas)
+  if(ENABLE_BUNDLED_OPENBLAS)
+    add_dependencies(p4est openblas)
+  endif()
```

Comment:
When OpenBLAS is loaded from a prefix, there is no `openblas` target to add as a dependency. I added this workaround, but is this the way to do it (then a similar condition should wrap `zlib`) or should this case be implemented differently?

Reply:
I don't think p4est needs openblas though, open d4est and petsc, but I added an add_dependecy to petsc.cmake I think, so perhaps this isn't needed. Did you run into issues when you tried a prefix install?